### PR TITLE
Sign in outage messages

### PR DESCRIPTION
**Pull request by annaswims**
## Description
Add outage messages for id.me MHV and MVI

## Testing done
My first react PR 😬

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by U-DON — APPROVED** [review]
LGTM, but be sure to correct the single apostrophes to the curly ones. It looks like there are some left over in the MHV copy here, so just use those consistently.

File: src/platform/user/authentication/components/SignInModal.jsx [hunk 1]
```diff
@@ -57,27 +72,30 @@ class SignInModal extends React.Component {
...
+          [externalServices.mhv],
+          'You may have trouble signing in with My HealtheVet',
+          'warning',
+          "We’re sorry. We’re working to fix some problems with our My HealtheVet sign in process. If you'd like to sign in to VA.gov with your My HealtheVet account, please check back later.",
+        )}
+        {this.downtimeBanner(
+          [externalServices.mvi],
+          'You may have trouble signing in or using some tools or services',
+          'warning',
+          "We're sorry. We're working to fix a problem that affects some parts of our site. If you have trouble signing in or using any tools or srevices, please check back soon.",
```
Comment (U-DON) [inline src/platform/user/authentication/components/SignInModal.jsx]: Typo: `srevices` -> `services`.

**Comment by narin** [comment]
@U-DON I think I recall a new session expired page being created by you in a separate ticket, is that correct?  @BBarone1 If so, is the work in this ticket meant to redirect to that new page, or just display this message on the current `/auth/login/callback/` page where we display our other error codes?

**Review by narin — COMMENTED** [review]
Fixed apostrophes, typos, and merge conflicts. LGTM.

**Pull request merged**